src/recipe/recipe.controller.ts

```diff
@@ -73,6 +75,7 @@ export class RecipeController {
     return this.recipeService.findAllRecipes(query);
   }
 
+  @UseInterceptors(CacheInterceptor)
```

Comment:
Please check what it does and why it's here.
Please refer to [this article](https://www.tomray.dev/nestjs-caching-redis) 
and [nest documentation](https://docs.nestjs.com/techniques/caching) about caching. Also pay attention to the notes about TTL using cache-manager